packages/react-accordion/src/components/Accordion/useAccordion.ts

```diff
@@ -23,14 +22,11 @@ export const useAccordion = (
   ref: React.Ref<HTMLElement>,
   defaultProps?: AccordionProps,
 ): AccordionState => {
-  const navigationAttributes = useArrowNavigationGroup({ circular: true });
```

Comment:
PR title says "disabled state styling", can you please clarify why it is removed?

Reply:
Because there's an entira new PR to solve A11y #17668, so I'm just removing the remanescants of the previous implementation. I'll to the description of this PR about that.